md/user-application-case-list.md

```diff
@@ -0,0 +1,136 @@
+  
+Less obvious difference:
+Users can add a comment in an open case, but cannot anymore in an archived case.  
+Comments are discribed a little below in this page.
+
+### Case information
+Each case has been rendered in the form of a container, with the following information:
+  * Case ID. It is clickable to navigate to the [case details](#case-details)
+  * Process name (version). The process and version that has been started to give way to this instance
+  * Started by. The initiator of the case
+  * Start date. The date and time this case has been instantiated. It is displayed in the format of the loggued user locale or chosen application language.
+  * Archived cases only: End date
+  * Open cases only: Available tasks. Number of tasks the loggued user can do in this case at that very moment in the case lifetime (in the "To do" or "My tasks" basket)
+  * "..." icon, to navigate to the [case details](#case-details)
+  * If available for the case, the next row displays the case search keys (between 1 and 5)
+
+### Filter by process name and version
+Open and Archived case lists can be filtered by process name and version, in the "All processes (All versions)" drop down list.
+
+### Filter by initiator
+Open and Archived case lists can be filtered by the initiator of the case. By default, the lists display the cases started by anyone.  
+By checking the "Started by me" checkbox, they are filtered by the loggued user as the initiator.  
+To know more about how to set an entity of the organization as the initiator of a case, go to the [actors](actors.md) page, or check the [User process list](user-process-list) page.  
+  
+### Refresh
+As time goes by, cases evolve by the action of other people or systems and the user needs an up-to-date vision of this status.
+The "loop" icon, at the right of the filter by initiator, allows the user to refresh the case list. 
+
+### Search by process name
+In case the number of processes is very big, the user can also search by process name. This search will then act as a filter by process name.
+
+### Search by search keys
+::: info
+This feature has been made available in the Community edition since Bonita 2021.1. It used to be only available in the Enterprise edition.
+:::
+
+#### Search keys
+
+Search keys are data attached to the process definition to distinguish between cases of the process. For example: the customer's name for a request.  
+It eases the search for a case if case Id is not what users favor when referring to a case.  
+They are instantiated through operations, mainly after the instantiation form has been submitted and the case starts.  
+They can be updated anytime during the lifetime of the case.  
+Search keys are optional and it is possible to define up to five search keys for a process.  
+All available search keys are displayed for a given case.  
+Technical note: A search key is translated by a database index in Bonita Engine.  
```

Comment:
same comment as above: I think it's more like there is a database index on each search index column. The word translated is misleading for me

Reply:
yes